infrastructure/README.md

```diff
@@ -49,6 +49,24 @@ To connect from SDK or REST we need to know the network address and authorizatio
     cat $ALGORAND_DATA/algod.net
     cat $ALGORAND_DATA/algod.token
 
+## Using Sandbox 2.0
+
+Algorand Sandbox is a fast way to create and configure an Algorand development environment with [Algod](https://github.com/algorand/go-algorand) and [Indexer](https://github.com/algorand/indexer). To quickly bring up a private network with algorand sandbox and use it within algob, following jobs are provided (in `/infrastructure`) :-
+1. `sandbox-up` - Clones the sandbox [git repo](https://github.com/algorand/sandbox.git) in `~/sandbox` and brings up the network (this might take a while).
```

Comment:
```suggestion
1. `sandbox-up` - Clones the sandbox [git repo](https://github.com/algorand/sandbox.git) in `~/sandbox` and fetched the network (this might take a while).
```

Reply:
what do you mean by bring up and bring down ?

Reply:
sandbox-up is essentially doing `docker-compose up` which sets up and brings up the docker containers for `algod`, `indexed` and `postgres` (indexer-db). Screenshot attached. 
![image](https://user-images.githubusercontent.com/33264364/106601031-2de37500-6581-11eb-8b48-c8bc3cd2ce13.png)

I don't think `fetches` would be the right word to replace

Reply:
Agree, _fetches_ is not the right word here. How about _setups_: 
_and setups the network_

Reply:
Yes, updating to: *setups the networks*